Shaders/PostProcess/ScreenSpaceAmbientOcclusion/private/SSAO_ComputeAmbientOcclusion.fx

```diff
@@ -33,7 +33,7 @@ float3 SampleNormalWS(float2 ScreenCoordUV)
 
 float SamplePrefilteredDepth(float2 ScreenCoordUV, float MipLevel)
```

Comment:
The function name is now misleading - it returns Z, not depth.
Please rename this function and others to reflect what they actually do, or alternatively, convert depth to z where the return value is used.

Reply:
@TheMostDiligent 
The function returns a hyperbolic depth in NDC space, not a linear depth. It is only necessary for OpenGL